app/javascript/components/admin/UserModal.js

```diff
@@ -12,12 +12,12 @@ class UserModal extends React.Component {
       roleTitle: this.props.initialUserData.role_title ? this.props.initialUserData.role_title : this.props.roles[0],
       isAPIEnabled: this.props.initialUserData.is_api_enabled ? this.props.initialUserData.is_api_enabled : false,
       isLocked: this.props.initialUserData.is_locked ? this.props.initialUserData.is_locked : false,
+      notes: this.props.initialUserData.notes ? this.props.initialUserData.notes : '',
     };
   }
 
-  handleEmailChange = e => {
-    const val = e.target.value;
-    this.setState({ email: val });
+  handleChange = event => {
+    this.setState({ [event.target.name]: event.target.value });
```

Comment:
can you make this `event?.target?.value || null` just in case it's ever empty?

Reply:
Fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/9e3e31175674140efbeb877b6bbbf711cb09768f